ui/src/app/pages/data/data-set/genomic-extraction-modal.tsx

```diff
@@ -41,6 +79,18 @@ export const GenomicExtractionModal = ({
         </span>
       </TextColumn>
     </ModalBody>
+    {runningExtract &&
+      <WarningMessage iconSize={30} iconPosition={'top'} data-test-id='running-extract-warning'>
+        An extraction is currently running for this dataset; it was started {TimeAgoWithVerboseTooltip(runningExtract.submissionDate)}
+      </WarningMessage>
+    }
+    {!runningExtract && succeededExtract &&
+      <WarningMessage iconSize={30} iconPosition={'top'} data-test-id='preexisting-extract-warning'>
```

Comment:
nit: can we collapse these two elements so that there's only one code block with `<WarningMessage>`? IMO, that makes it more clear that only one of them should ever exist and one overrides the other.